lib/logstash/outputs/http.rb

```diff
@@ -80,6 +81,9 @@ class LogStash::Outputs::Http < LogStash::Outputs::Base
   # Otherwise, the event is sent as json.
   config :format, :validate => ["json", "form", "message"], :default => "json"
 
+  # Set this to true if you want to enable gzip compression for your message
+  config :compression, :validate => :boolean
```

Comment:
We also need to update the static doc at https://github.com/logstash-plugins/logstash-output-http/blob/master/docs/index.asciidoc with that new option so it will appear on our website.